cdc/entry/mounter.go

```diff
@@ -150,16 +147,18 @@ func (m *mounterImpl) collectMetrics(ctx context.Context) {
 	changefeedID := util.ChangefeedIDFromCtx(ctx)
 	metricMounterInputChanSize := mounterInputChanSizeGauge.WithLabelValues(captureAddr, changefeedID)
 
+	metricsTimer := time.NewTimer(defaultMetricInterval)
```

Comment:
defer stop()

Reply:
NewTimer or NewTicker ?

Reply:
I prefer Timer because Ticker buffers timeout internally that may lead to record metrics twice in a very short period.